packages/api/portfolioDrafts/getPortfolioDraft.ts

```diff
@@ -0,0 +1,50 @@
+import { GetCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { ErrorCodes } from "../models/Error";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "../utils/response";
+import { PortfolioDraft } from "../models/PortfolioDraft";
+import { isPathParameterPresent } from "../utils/validation";
+import { dynamodbDocumentClient as client } from "../utils/dynamodb";
+import { DATABASE_ERROR } from "../utils/errors";
+
+export const NO_PORTFOLIO_PATH_PARAM = new ErrorResponse(
+  { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path." },
+  ErrorStatusCode.BAD_REQUEST
+);
+export const NO_SUCH_PORTFOLIO = new ErrorResponse(
+  { code: ErrorCodes.INVALID_INPUT, message: "Portfolio Draft with the given ID does not exist" },
+  ErrorStatusCode.NOT_FOUND
+);
+
+/**
+ * Gets a detailed view of a Portfolio Draft
+ *
+ * @param event - The GET request from API Gateway
+ */
+export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+
+  // TODO: validate that we get a valid ATAT_TABLE_NAME env variable
+  if (!isPathParameterPresent(portfolioDraftId)) {
+    return NO_PORTFOLIO_PATH_PARAM;
+  }
+
```

Comment:
After checking for presence, suggest checking param is a valid UUIDv4 and issuing PATH_VARIABLE_INVALID if necessary.

Reply:
After team discussion, checking the param for validity would be fine for the purposes of avoiding a database call, but not for returning information in the response.